iscript/src/iscript/constants.py

```diff
@@ -14,6 +14,7 @@
         "skip_dirs": tuple(),
         "zipfile_cmd": "zip",
         "create_pkg": True,
+        "hardened_runtime_only_files": "geckodriver",
```

Comment:
I was wondering if we might break something for mac Firefox signing, but [looking at the code](https://github.com/mozilla-releng/scriptworker-scripts/blob/9946f96ac5dc203b096c66dc76886447a0d0d2d3/iscript/src/iscript/mac.py#L175) we're probably good.

Reply:
I think so too.